packages/venia-concept/src/components/LoadingIndicator/indicator.js

```diff
@@ -2,8 +2,8 @@ import React, { Component } from 'react';
 
 import classify from 'src/classify';
 import defaultClasses from './indicator.css';
-// TODO: move to a common location
-import logo from '../Header/logo.svg';
+
+import logo from '../Logo/logo.svg';
```

Comment:
I know this logo is used elsewhere in the app (even though it was exported from `Header`). Make sure nothing else was broken by this move.